src/ansys_sphinx_theme/static/404.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+Oops!
+=====
+This is unexpected. The page you are requesting does not exist.
+
+If this page should exist, please contact `pyansys.support@ansys.com <mailto:pyansys.support@ansys.com>`_.
```

Comment:
I am not sure if there is a support for Ansys related projects. All public projects within Ansys organization contain references to personal email or PyAnsys support.

Reply:
Honestly, this is out of the PyAnsys org. Anybody could potentially use this Sphinx theme for their own local projects. So, I'd recommend to not provide any support email. In fact, I'd suggest trying to link somehow to the Issues page of the GitHub repo associated to the build. For example, if PyMAPDL docs are using the Sphinx theme, this should link to the Issues page of PyMAPDL. I don't know if it'd be possible, but that would be the ideal scenario.

Reply:
From private discussion, let us investigate the `setup(app)` function to extract the `github_url` variable from the `html_theme_options`.

Reply:
I forgot about this part... It turns out that we can use Jinja syntax `{{ theme_github_url }}` 😍

Reply:
That's just great! Thanks @jorgepiloto